internal/pkg/cli/app_delete.go

```diff
@@ -225,6 +238,67 @@ func (o *deleteAppOpts) deleteSvcs() error {
 	return nil
 }
 
+func (o *deleteAppOpts) deleteTasks() error {
+
+	envs, err := o.store.ListEnvironments(o.name)
+	if err != nil {
+		return fmt.Errorf("list environments for application %s: %w", o.name, err)
+	}
+	o.envs = envs
+
+	// Delete tasks from each environment (that is, delete the tasks that were created with each environment's manager role)
+	var envTasks []deploy.TaskStackInfo
+	for _, env := range envs {
+		envSess, err := o.sessProvider.FromRole(env.ManagerRoleARN, env.Region)
+		if err != nil {
+			return err
+		}
+		envCF := o.cfn(envSess)
+		envECR := o.ecr(envSess)
+		envTasks, err = envCF.GetTaskStackInfo(o.name)
+		if err != nil {
+			return fmt.Errorf("get tasks deployed in environment %s: %w", env.Name, err)
+		}
+		for _, t := range envTasks {
+			o.spinner.Start(fmt.Sprintf("Deleting task %s from environment %s.", t.TaskName(), env.Name))
+			if err := envECR.ClearRepository(t.ECRRepoName()); err != nil {
+				o.spinner.Stop(log.Serrorf("Error emptying ECR repository for task %s\n", t.TaskName()))
+				return fmt.Errorf("empty ECR repository for task %s: %w", t.TaskName(), err)
+			}
+			if err := envCF.DeleteTask(t); err != nil {
```

Comment:
`DeleteTask` deletes the CFN stack. Is it possible that there are still orphan tasks in the cluster after this? (like job delete problem)